### ISSUE TITLE
Allow to disable user and group management

### ISSUE THREAD

**Issue by azmeuk** [nextcloud/server#39431]
When user and group management is delegated to a third party service (LDAP, OIDC etc.), on might want to disable the user and group management feature in nextcloud.
A real life example that happened to me last week: I have a nextcloud installation using [nextcloud-oidc-login](https://github.com/pulsejet/nextcloud-oidc-login/). This plugins set the groups in a OIDC token, thus user group membership is always reset at login. A user complained to me that users kept leaving the groups he created with the nextcloud interface. I explained him that user and group management should be done in the OIDC server, but I would also disable the user and group management in the nextcloud interface to avoid future issues like this.

TL;DR: I suggest implementing a configuration option to enable/disable user and group management in nextcloud.
